lms/djangoapps/teams/views.py

```diff
@@ -58,7 +51,10 @@
 from .errors import AlreadyOnTeamInCourse, NotEnrolledInCourseForTeam
 
 
+
 # Constants
+from teams import is_feature_enabled
```

Comment:
Nit: the PyCharm refactor tool put this import in an odd location. Our guidelines suggest grouping imports categorically, and then I like to sort them alphabetically within a category.

See https://github.com/edx/edx-platform/wiki/Python-Guidelines